public/main.js

```diff
@@ -0,0 +1,200 @@
+let winCombo = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 5, 8], [0, 4, 8], [6, 4, 2]];
+let xLog = [];
+let oLog = [];
+let boardArray = [0, 1, 2, 3, 4, 5, 6, 7, 8]; // listing available spaces for AI 
+let moveCount = []; // for tracking location and number of moves
+let playerO = false;
+let count = 0; // for game timer
+let computerPlayer = null; 
```

Comment:
What if we made a change where we start "computerPlayer" = false, like the "playerO" ? Seems to wipe out a couple of the bugs. Now we're just left dealing with a few squares it gets confused on.